lib/gui/models/store.js

```diff
@@ -95,8 +95,8 @@ const storeReducer = (state, action) => {
 
         const drive = _.first(action.data);
 
-        // Even if there's no image selectected, we need to call `isDriveValid`
-        // to check if the drive is locked, and `{}` works fine with it
+        // Even if there's no image selectected, we need to call several
```

Comment:
I've never used it myself, but in other repos I've seen people submitting typo PRs found by a tool called [codespell](https://github.com/lucasdemarchi/codespell).

(And of course just to nitpick, that fixes spelling, not grammar ;) )